patches/react-native+0.69.3.patch

```diff
@@ -0,0 +1,163 @@
+diff --git a/node_modules/react-native/Libraries/Blob/BlobManager.js b/node_modules/react-native/Libraries/Blob/BlobManager.js
+index 0d4cee7..9e39534 100644
+--- a/node_modules/react-native/Libraries/Blob/BlobManager.js
++++ b/node_modules/react-native/Libraries/Blob/BlobManager.js
+@@ -14,6 +14,7 @@ const BlobRegistry = require('./BlobRegistry');
+ import type {BlobData, BlobOptions, BlobCollector} from './BlobTypes';
+ import NativeBlobModule from './NativeBlobModule';
+ import invariant from 'invariant';
++import { getBlobForArrayBuffer } from 'react-native-blob-jsi-helper';
+ 
+ /*eslint-disable no-bitwise */
+ /*eslint-disable eqeqeq */
+@@ -69,9 +70,7 @@ class BlobManager {
+         part instanceof ArrayBuffer ||
+         (global.ArrayBufferView && part instanceof global.ArrayBufferView)
+       ) {
+-        throw new Error(
+-          "Creating blobs from 'ArrayBuffer' and 'ArrayBufferView' are not supported",
+-        );
++        return getBlobForArrayBuffer(part);
+       }
+       if (part instanceof Blob) {
+         return {
+diff --git a/node_modules/react-native/Libraries/Blob/FileReader.js b/node_modules/react-native/Libraries/Blob/FileReader.js
+index c58dfba..2f3aa15 100644
+--- a/node_modules/react-native/Libraries/Blob/FileReader.js
++++ b/node_modules/react-native/Libraries/Blob/FileReader.js
+@@ -10,6 +10,7 @@
+ 
+ const Blob = require('./Blob');
+ const EventTarget = require('event-target-shim');
++const { toByteArray } = require('base64-js');
```

Comment:
why not react-native-quick-base64 here? they have converters for that

Reply:
Good catch! I'm glad I updated this patch in this PR haha 🤣